lxd-migrate/main_migrate.go

```diff
@@ -566,7 +566,7 @@ func (c *cmdMigrate) Run(cmd *cobra.Command, args []string) error {
 
 	defer func() {
 		if !success {
```

Comment:
Good place for using the `revert` package here, but probably outside of scope of this change :)

Reply:
Done :+1: .